File: estruturas_de_dados_e_algoritmos/1 - topicos iniciais/python/readjson/read-file.py
import json
from dateutil import parser

# pip install python-dateutil

with open('file.json', 'r') as file:
    jsonData = json.load(file)

def parse_timestemp(timestemp):
    return parser.isoparse(timestemp)


for course in jsonData:
    print(f"Curso: {course['title']}")

    for lesson in course['lessons']: 
        print(f"    Aula {lesson['id']} - {lesson['title']}")
        print(f"        Media: {lesson['media']}")
        print(f"        timestemp: {parse_timestemp(lesson['timestamp'])}")

File: estruturas_de_dados_e_algoritmos/1 - topicos iniciais/javascript/readjson/read-file.js
import { jsonData } from './file.js';

const parseTimestamp = (timestamp) => {
  return new Date(timestamp);
};

for (const course of jsonData) {
  console.log(`Curso: ${course.id} - ${course.title}`);

  for (const lesson of course.lessons) {
    console.log(`   Aula: ${lesson.id}: Title: ${lesson.title} `);
    console.log(`       Media: ${lesson.media}`);
    console.log(`       Timestamp: ${parseTimestamp(lesson.timestamp)}`);
    console.log();
  }
}
